docs/pages/install/docker.mdx

```diff
@@ -140,3 +140,38 @@ It is recommended that admins bind a Pelican configuration file on the host mach
 
 
 4. You should note that your running Pelican origin server is logging debug level messages.
+
+## Additional Configurations for Running Pelican in a Container
+
+### Server hostname and port
+
+Pelican server has a built-in web server to handle various API requests. By default, it uses server hostname and a port number as the server address (e.g. `https://127.0.0.1:8444`). In a container environment, the hostname is an alphanumeric value, e.g. `6ee28c5df997`, and the corresponding Pelican server address is `https://6ee28c5df997:8444`. This value is for container internal communications and is not accessible from outside of the container. In local deployment, to access the container ports from the host machine, one can publish a port from inside the container to the host machine.
+
+```bash
+docker run -p 8444:8444 origin
+```
+
+However, Pelican server still recognize `6ee28c5df997` as its hostname, causing a mismatch.
+
+In practice, there is usually a reverse-proxy service (such as Nginx, Traefic, etc.), to direct traffic from a human-readble domain name (e.g. https://example-origin.org) to your container (e.g. `6ee28c5df997:8444`). If Pelican server doesn't catch this information and uses it as its server address, there will be a mismatch when the server advertise itself to other Pelican services.
+
+Therefore, to keep things consistent, Pelican internally should use the same human-readble domain name as its web server's _external address_. To do so, configure `Server.ExternalWebURL` to the domain name the outside world uses to access the Pelican web service, e.g. https://example-origin.org:8444 for a production service or https://localhost:8444 for a local deployment.
+
+Notice that there is a port number, `8444`, following the hostname in the URLs above. This is the default port for Pelican web server. When publishing ports from the container, you may choose a different port on the host machine to publish to. Two convential options are `443` or `8080`, which are default ports for a web service. If you publish `8444` port from the container to `443`, or `8080` on the host machine, i.e. `docker run -p 8444:443`, you may remove the port number from `Server.ExternalWebURL`, i.e. `https://example-origin.org`. For other ports, you need to leave the port number in the URL.
+
+## Troubleshooting
+
+### Error running Pelican Docker image on macOS with a M-series chip
+
+When running Pelican Docker image on macOS with a M-series chip, you may get the following error message:
+
+```console
+docker: no matching manifest for linux/arm64/v8 in the manifest list entries.
+See 'docker run --help'
+```
+
+This is because Pelican Docker images are built for Linux/AMD64 architecture only. To fix the issue, run the image with `--platform linux/amd64` instead:
```

Comment:
Let's add a note that this isn't a great idea for production servers because the x86 emulation makes things much less efficient.